go.mod

```diff
@@ -27,6 +27,8 @@ require (
 
 require go.uber.org/zap v1.19.1
 
+require github.com/ghodss/yaml v1.0.0
```

Comment:
I think we could put this (and zap) in the block above

Reply:
actually, is there a particular reason for using _this_ package instead of sigs.k8s.io/yaml v1.3.0 which we already import or https://github.com/go-yaml/yaml which is pretty much the standard? this one seems quite unmaintained

Reply:
I changed this to use the more standard package, as suggested, and fixed the go.mod file